plugins/processors/traffic_shaper/traffic_shaper_test.go

```diff
@@ -0,0 +1,95 @@
+package traffic_shaper
+
+import (
+	"math"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/metric"
+	"github.com/influxdata/telegraf/testutil"
+)
+
```

Comment:
Please include a test with tracking metrics. See the other processors for examples.